src/test/java/ua/rent/masters/easystay/service/impl/AccommodationServiceImplTest.java

```diff
@@ -70,18 +67,20 @@ void save_ValidAccommodationRequestDto_ReturnsAccommodationResponseDto() {
         // Given
         AccommodationRequestDto requestDto = createAccommodationRequestDto();
         Accommodation accommodation = createAccomadation();
+        Accommodation savedAccommodation = createAccomadation();
```

Comment:
You are really needed the same var twice?
```suggestion
        Accommodation accommodation = createAccomadation();
```